airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -25,9 +25,13 @@
 from __future__ import annotations
 
```

Comment:
> I thought we also might add/merge config in AwsCredentialHelper rather than in hook

What would be the advantage of adding it there instead?

Reply:
Right now AwsCredentialHelper resolve connection configurations between Hook args and Connection parameters.
So I assume that a good place to keep it there

Reply:
Ohhh... and I just figure out that amazon secrets backends do not use `AwsBaseHook` and directly use helper and session factory

Reply:
Sorry for the wait, I wanted to think this through and make sure I understood the implications.  If I am understanding correctly: doing it the way it is currently written means that we do not get any data from when Airflow itself connects to SecretsManager.  That was somewhat accidental, but I consider that an absolute win.  I/We don't want to be "snooping" on traffic where Airflow is retrieving secrets.  The main goal here is to understand which services and endpoints are being used in DAGs so we can prioritize adding new hooks and operators accordingly.

Reply:
@Taragolis What do you think?   Is this alright or do you still want that change?

Reply:
@ferruzzi Sorry miss notification.

This more about preventing circular imports. Right now if user try to obtain _config_  from Secret Backend ([with `_secret` suffix](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html#setting-configuration-options)) it actually happen before some airflow modules properly imported and in this case local import might not help.

It should not affect if user get _variables_ and _connections_ from Secrets Backends.

That mean right now it is fine that this imports inside methods of AwsGenericHook/AwsBaseHook